lib/src/data/datasource/remote/api_service.dart

```diff
@@ -0,0 +1,17 @@
+import 'package:hearthstoneapp/src/core/util/constants.dart';
```

Comment:
relative import

Reply:
My IDE doesn't let me change to relative imports automatically using the context menu, so I was told it was ok leaving them like that.